### DISCUSSION TITLE
Heroku Crash

### DISCUSSION THREAD

**Discussion by Articial** [eritislami/evobot#900] [Q&A] [answered]
i have this error on Heroku. when i clone the repo to my local machine, the "config.json" file didn't exist. i have try to clone and add the config.json again and commit it again but it's still show the same error
![image_2021-09-18_124316](https://user-images.githubusercontent.com/80366906/133877477-8b99f85f-95d9-48be-abac-8a94efc7ea77.png)

**Comment by eritislami** [comment, marked as answer]
You need to add the config values into heroku config variables
![](https://devcenter1.assets.heroku.com/article-images/321-imported-1443570183-321-imported-1443554644-389-original.jpg)
TOKEN
YOUTUBE_API_KEY
...others

**Reply by Articial** [reply]
Thank You so much for the Reply! I actually figured that out too after i post the question but i forgot to confirm on this Discussions.
